build.sh

```diff
@@ -4,6 +4,9 @@ set -e
 
 ORG_PATH="github.com/appc"
 REPO_PATH="${ORG_PATH}/docker2aci"
+# TODO: use "git describe --dirty" when we actually have tags pushed
+VERSION=$(git rev-parse --short HEAD)
+GLDFLAGS="-X github.com/appc/docker2aci/lib.Version=${VERSION}"
```

Comment:
I don't think we need to care about go1.4

How about I set the default version in `lib/version.go` to `0.9.0` and `0.9.0+git` like we do in rkt (with a bump-version script), but if you happen to build with `./build.sh` you also get the git commit with `git describe`?